### PR TITLE
Remove NeoTheology access cringe

### PR DESCRIPTION
**Pull request by SirRichardFrancis**
## About The Pull Request

Nuked hydroponics and old church access, NT access reduced to 2 types - preacher and disciple .
Access on the map and lockers changed accordingly.
Preacher now actually able to give/take church access.
Moved NT access from cruciforms to ID cards. That is neat feature, but since we have no special litanies/angels to work with that - it should go for now.
Exception for inquisitor, who will get additional access to cruciform on spawn.

image

## Why It's Good For The Game

Fixes cool.

## Changelog
:cl:
del: removed old church-related access
tweak: modified NT access to make sense
/:cl:

### CODE REVIEW

**Comment by heymando** [comment]
pog

**Comment by vode-code** [comment]
If all this PR did was replace the Crematorium access needed on church with Disciple access, this would have been a good PR.
If this worked properly, Disciples from the Last Shelter, Inquisitors, and roundstart unofficial NT would be able to access Church areas, allowing for Disciples to actually contribute well. This PR almost allowed it, but then ruined it by removing Disciple access from Cruciforms.

**Comment by vode-code** [comment]
As I see it, Disciples should have access to all the doors that were previously labeled Crematorium, Preacher should replace Chapel Office, Agrolyte should replace Hydroponics, Custodian should replace Janitor, and Acolyte access should replace whatever access is currently being used for Acolyte stuff. This would allow for Disciples to more effectively contribute, without allowing NT Vagabonds to raid job-specific areas.

**Comment by garrulousTeamotei** [comment]
The Preacher already has the means to add/remove access. They already start with the 'NT Change ID' access on their ID. 
And, isn't one of the main reasons of keeping ID tied to the Cruciform so a resurrected player isn't kneecapped due to a loss of all ID? Outside of looting corpses and breaking into specific command offices, It's a pain to grab a spare ID.

**Comment by heymando** [comment]
perhaps it is more worth to instead let this pass and rework the idea into something that is more palatable to play with. for example, it occurs to me that the church has completely normal doors and there's no logical reason why access is tied to cruciforms (but ONLY for people who join directly as church roles) and with no other way to interact with the concept in-game. instead, it may be more enjoyable to relegate cruciform access to prayers, with higher-ranking members able to give progressively higher and arbitrary ranks of access, and have cruciform access only allow the opening of special religious doors created using the construction litany.

tl;dr: improve it on your own afterwards if it's such a crucial idea 
food for thought

**Comment by vode-code** [comment]
> 
> 
> The Preacher already has the means to add/remove access. They already start with the 'NT Change ID' access on their ID.
> And, isn't one of the main reasons of keeping ID tied to the Cruciform so a resurrected player isn't kneecapped due to a loss of all ID? Outside of looting corpses and breaking into specific command offices, It's a pain to grab a spare ID.

1. Inquisitors don't, and they can convert.
2. Cruciforms without Crematorium access also happen from conversion, roundstart cruciforms, and cruciforms made by The Last Shelter.

> there's no logical reason why access is tied to cruciforms (but ONLY for people who join directly as church roles) and with no other way to interact with the concept in-game.

If the doors had Disciple access instead of Crematorium access, then it would allow anyone with a cruciform to use church doors, not just church roles.
Sadly, while this PR makes church doors have Disciple access, it also removes Disciple access from non-inquisitor cruciforms.

**Comment by SirRichardFrancis** [comment]
Ideally, we need means of modifying access stored on cruciform, so new churchies get it after a short litany. But we don't have that.
Maybe I just understand concept of NT wrong, but they seemed like a close-knit religious commune, and having 5 different access types for a little ship chapel, eh? If you just ask your fellow NT to give you whatever from their locked area - they will give.
Furthermore, most of agrolyte stuff is free for the taking in nearby crates, vendors and hydroponic locker, and the rest of "job-specific" stuff can be easily crafted or made in bioprinter.
From my experience, having job-locked areas inside NT is just an inconvenience. Used ritual room and now it's all bloody? Nah, you can't take cleaning stuff from locker 4 tiles away, go make your own.

I've played as Last Shelter disciple last week and preacher complained about 'NT Change ID' being broken.
Absence of spare ID's in the chapel is a good point tho, gonna add one box to preacher's locker.
If someone stole ID from churchie - that's their problem, ask preacher/FO/captain for a new one.
Church also unable to fire/exile a troublesome disciple, they will retain access no matter what.

Why would you want hobos inside the church tho? To steal your shiny holy sword an bioprinter discs?

**Comment by SirRichardFrancis** [comment]
image

**Comment by vode-code** [comment]
> Church also unable to fire/exile a troublesome disciple, they will retain access no matter what.
> 
> Why would you want hobos inside the church tho? To steal your shiny holy sword an bioprinter discs?

conflict inside NT could help make Inquisitors have more inquisition to do, instead of just arranging crusades once every twenty rounds.

**Comment by SirRichardFrancis** [comment]
> conflict inside NT could help make Inquisitors have more inquisition to do, instead of just arranging crusades once every twenty rounds.

So we just hope for an antag to spawn and come kill/torture a troublemaker? Why not just have working doors instead?

**Comment by SirRichardFrancis** [comment]
> inquisitors aren't all that rare, and if we don't have an inquis, there isn't anything stopping a regular member from taking action.
> plus, stealing the sword and disks isn't hard already, there's plenty of windows to break, and so if nothing other than difficulty getting there was stopping them, they'd have already been constantly doing it.

So we just give vagas access to all of the valuable church items and machinery, while locking away soap and hoes? Don't take it personally, but isn't that just fucking stupid?
Sure is, they can break glass/door/wall and get in, but that can be said about literally every other departments. Why not give vagas access to armory/lab/engine then? And kill them if they took something. Same thing.

**Comment by vode-code** [comment]
> So we just give vagas access to all of the valuable church items and machinery, while locking away soap and hoes? Don't take it personally, but isn't that just fucking stupid?

honestly I don't care too much about the custodian and garden and whatever accesses, it just feels odd to not have them.
> Sure is, they can break glass/door/wall and get in, but that can be said about literally every other departments. Why not give vagas access to armory/lab/engine then? And kill them if they took something. Same thing.

Vagabonds already break into armory, chemistry, and R&D on a regular basis without access, and if there is no techno, they may break into engine to start it as well. This wouldn't give access to anyone without a cruciform, and one of the main benefits of a cruciform is that it means the church is on your side, which is something you can easily lose. Furthermore, a church-goer has no motivation to steal anything other than the Sword Of Truth, as there is no real reason to steal NT disks instead of just using them especially since normal autolathes can't use biomatter and you can't steal NT machinery; the Sword Of Truth gets stolen even now, and actually isn't impossible to steal midgame without a cruciform. The only downside I can see here is that people will have to be more careful with forced converts as it isn't actually brainwashing like excels are, just an ultimatum favoring the church, and having to keep an eye on that guy you force converted isn't actually a bad thing anyways.

**Comment by vode-code** [comment]
only cruciformed vagabonds would be able to benefit from this, and cruciformed vagabonds cannot become non-cruciformed vagabonds.

**Comment by Iglpigl** [comment]
> only cruciformed vagabonds would be able to benefit from this, and cruciformed vagabonds cannot become non-cruciformed vagabonds.

To be fair they can utilizing bug abuse, but otherwise you are right, yes.

**Comment by TheShown911** [comment]
May conflict with https://github.com/discordia-space/CEV-Eris/pull/6420.

**Comment by SirRichardFrancis** [comment]
> May conflict with #6420.

Totally conflicts with #6420. Not my fault, this PR is 5 days older than the other.

**Comment by garrulousTeamotei** [comment]
> 
> 
> I still think that giving vagabonds NT access is a bad idea, not changing that.

Speaking from a good deal of experience here. On at least a dozen of occasions, as both a Preacher and as an acolyte who gained 'NT Change ID' access, giving Crematorium access to NT vagabonds and other NT players has never led to any negative repercussions, at least not by my knowledge. No Relics ever went missing, resources plundered, or things broken. 
Funny enough, the only occasion of a NT Relic going missing (Other than being stolen by someone outside the church) was where the Preacher sold the Sword of truth for a Junk shovel with some good tool mods on it.

**Comment by SirRichardFrancis** [comment]
> Speaking from a good deal of experience here. On at least a dozen of occasions, as both a Preacher and as an acolyte who gained 'NT Change ID' access, giving Crematorium access to NT vagabonds and other NT players has never led to any negative repercussions, at least not by my knowledge. No Relics ever went missing, resources plundered, or things broken.
> Funny enough, the only occasion of a NT Relic going missing (Other than being stolen by someone outside the church) was where the Preacher sold the Sword of truth for a Junk shovel with some good tool mods on it.

Nothing stops you from giving vags that access. I'm not giving it from roundstart. Moebius, Guild and Technomancers can be cruciformed too. In case of a faction conflict they'll just walk in.

**Comment by hyperioo** [comment]
@SirRichardFrancis If the pull request was closed due to map conflict (since it was made for previous church layout), I can make a map PR that changes door accesses to be consistent your PR. If you remove your map changes it should not conflict anymore right?
